test/extended/operators/clusteroperators.go

```diff
@@ -45,12 +43,10 @@ var _ = g.Describe("[sig-arch] ClusterOperators [apigroup:config.openshift.io]",
 		"support",
 	)
 
+	oc := exutil.NewCLIWithoutNamespace("clusteroperators")
```

Comment:
Would you please mention in the PR description why `NewCLIWithoutNamespace` is more preferable than `NewCLI`?

Reply:
Cluster operator tests dont require a namespace, all checks are performed against non namespaced objects.